NeoML/src/TraditionalML/KMeansClustering.cpp

```diff
@@ -671,6 +776,21 @@ void CKMeansClustering::selectInitialClusters( const CDnnBlob& data, int seed, C
 	}
 }
 
+static void vectorCopy( const CKMeansClustering& owner, IMathEngine& mathEngine, int vectorSize, const CFloatHandle& firstHandle, const CConstFloatHandle& secondHandle )
+{
+	if( vectorSize / /*align*/16 < MinOmpOperationCount ) {
```

Comment:
`16` is used in multiple places -> `constexpr vectorCopyAlignment`

Reply:
Also `IsOmpRelevant` doesn't divide by alignment

Reply:
fixed